src/controllers/TalksController.php

```diff
@@ -822,16 +788,46 @@ public function removeSpeakerForTalk(Request $request, PDO $db)
             throw new Exception("There was a problem assigning the talk", 500);
         }
 
-        //If we are unit testing, then we can't exit or send headers!
-        if (defined('UNIT_TEST')) {
```

Comment:
adjusted these to match @heiglandreas  change. His was merged while this pull request was open.